reactioncommerce/reaction
# [WIP] Fix selecting filtered orders not working properly

Resolves #2837

This PR fixes a bug on the orders page where when a filter is applied, `select all` selects all orders present in the table, including those not relevant to the filter.

#### Test Instructions
- Select the orders from the side nav bar.
- Select a filter and click the select all Checkbox, you will observe it displays the actual number of selected items.
